server/src/main/java/org/elasticsearch/features/FeatureData.java

```diff
@@ -50,7 +48,8 @@ public static FeatureData createFromSpecifications(List<? extends FeatureSpecifi
                     );
                 }
 
-                if (hfe.getValue().after(CLUSTER_FEATURES_ADDED_VERSION)) {
+                // FIXME
+                if (hfe.getValue().after(Version.CURRENT)) {
```

Comment:
Kinda expected, the 1 test failing is:
`FeatureServiceTests » testFailsNonHistoricalVersion`

Reply:
We should not have this as 8.14/current, we can bump it to 8.13 at most (here if we decide to ditch https://github.com/elastic/elasticsearch/pull/106296, in test only otherwise)

Reply:
That was just to get the yaml tests to run, obviously pending on above. But imho, I think we have to permit 8.14 to not block ourselves on other teams here.

Reply:
8.14 is current, I'd definitely _not_ like to permit 8.14 if we can avoid it

Reply:
You can revert also this change, so the PR can pass CI (and be approved and merged)

Reply:
thx, forgot about this one